packages/grid_client/src/modules/base.ts

```diff
@@ -231,8 +231,9 @@ class BaseModule {
       flist: data.flist,
       publicIP: await this._getMachinePubIP(deploymentName, deployments, data.network.public_ip),
       planetary: data.network.planetary
-        ? (workload.result.data as ZmachineResult).ygg_ip || (workload.result.data as ZmachineResult).planetary_ip
-        : "",
+        ? (workload.result.data as ZmachineResult).planetary_ip
+        : (workload.result.data as ZmachineResult).ygg_ip,
+      myceliumIP: data.network.mycelium.hex_seed ? (workload.result.data as ZmachineResult).mycelium_ip : "",
```

Comment:
this might crash if there is no mycelium on the network. so we are trying to access hex_seed on mycelium which is undefined
```suggestion
      myceliumIP: data.network.mycelium?.hex_seed ? (workload.result.data as ZmachineResult).mycelium_ip : "",
```